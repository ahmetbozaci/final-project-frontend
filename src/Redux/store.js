import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import logger from 'redux-logger';
import { composeWithDevTools } from 'redux-devtools-extension';
import rootReducer from './rootReducer';
import fetchDoctors from './doctor/doctorActions';

const store = createStore(
  rootReducer,
  composeWithDevTools(applyMiddleware(thunk, logger)),
);
store.dispatch(fetchDoctors());

export default store;
