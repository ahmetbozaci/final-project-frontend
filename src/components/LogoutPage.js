import React from 'react';
import { Link } from 'react-router-dom';
// import '../app.css';
import '../assets/style/logoutPage.css';

function Logout() {
  return (
    <section className="mainlogout">
      <div className="welcomelg text-center ">
        {/* <div className="hg3">
          <h1>Be HAPPY,</h1>
          <h1> Be HEALTHY</h1>
        </div> */}
        <Link to="login"><button type="button" className="btn btn-primary text-center first-page-button"> Start your health experience </button></Link>
      </div>
    </section>
  );
}

export default Logout;
