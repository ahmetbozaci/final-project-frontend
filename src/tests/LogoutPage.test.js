import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../Redux/store';
import LogoutPage from '../components/LogoutPage';

it('LogoutPage renders correctly', () => {
  const tree = renderer
    .create(<Provider store={store}><Router><LogoutPage /></Router></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
