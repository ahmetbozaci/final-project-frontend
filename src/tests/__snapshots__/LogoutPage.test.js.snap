// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutPage renders correctly 1`] = `
<section
  className="mainlogout"
>
  <div
    className="first-page-logo-div"
  >
    <a
      href="/login"
      onClick={[Function]}
    >
      <img
        alt="recovon logo"
        className="first-page-logo"
        src="recovon-logos-transparent.png"
      />
    </a>
  </div>
  <div
    className="logoutButton text-center"
  >
    <a
      href="/login"
      onClick={[Function]}
    >
      <button
        className="btn text-center first-page-button"
        type="button"
      >
         Start your health experience 
      </button>
    </a>
  </div>
</section>
`;
